### DISCUSSION TITLE
Field BT-36 (period)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#483] [Q&A] [answered]
The BT-36 (period) field in the eForms Notice Editor Demo application appears as a single field.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/af3cda70-9c1a-4d31-a6f5-973b38417bb6)

While in the Front eNotices 2 tool it appears with two fields.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/0108538b-86d5-4aad-b890-4862a83c23c1)

What is the difference between a form and another for the field? 
Could you send us a visual model with that field reported and completed as in the eNotices tool front part?

Thanks in advance

**Comment by rouschr** [comment, marked as answer]
I think currently eNotices2 is creating an extra field for durations, so they have special code to deal with this.
Month here is the unit and for currency and unit (if I remember correctly) we historically had no fields as those were considered implicit.
This is not correctly reflected in the editor demo and we are thinking about adding new fields for these to avoid having to need special code. I think that as there were other questions about similar cases we should raise the priority of this subject relative to other tasks.
NOTE: The SDK was initially built more for validation purposes (xsds, schematron, ...) and eNotices2 was doing the editor and using notice types which we took over in order to integrate with fields and nodes of the SDK so that impact of changes would be more integrated.
